tests/Configuration/LoaderTest.php

```diff
@@ -35,4 +36,15 @@ public function testLoadThrowsFileCannotBeParsedAsYamlExceptionWhenFileDoesNotCo
 
         $loader->load($file);
     }
+
+    public function testLoadThrowsParsedYamlIsNotAnArrayExceptionWhenFileDoesNotContainYamlThatCanBeParsedToAnArray(): void
+    {
+        $file = __DIR__.'/../../examples/does-not-contain-array.yml';
```

Comment:
IHMO in the example directory only working/executable examples should be stored. Instead of reusing the examples directory I would prefer having a `fixtures` or similar directory next to test file.

Reply:
Adjusted!

Reply:
Thanks :-)